commands/logs_e2e_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mmctl/client"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestlogsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Display single log line", func(c client.Client) {
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("number", 1, "")
+
+		data, err := testLogsCmdF(s.th.SystemAdminClient, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(data, 2)
+		s.Contains(data[1], "info app/plugin.go:223 Syncing plugins from the file store")
+	})
+
+	s.RunForAllClients("Display logs", func(c client.Client) {
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("logrus", true, "")
+		cmd.Flags().Int("number", 1, "")
+
+		data, err := testLogsCmdF(s.th.SystemAdminClient, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(data, 2)
+		s.Contains(data[1], "level=info msg=\"Syncing plugins from the file store\" caller=\"app/plugin.go:223\"")
+	})
+
+	s.RunForAllClients("Error when using format flag", func(c client.Client) {
```

Comment:
This case is not required since it is not related to the command itself. We can run this test with a regular user and see if they have permissions to see system logs.

Reply:
removed